cmd/tuple/changes.go

```diff
@@ -32,15 +33,32 @@ import (
 var MaxReadChangesPagesLength = 20
 
 func readChanges(
-	fgaClient client.SdkClient, maxPages int, selectedType string, continuationToken string,
+	fgaClient client.SdkClient, maxPages int, selectedType string, startTime string, continuationToken string,
 ) (*openfga.ReadChangesResponse, error) {
 	changes := []openfga.TupleChange{}
 	pageIndex := 0
 
+	var startTimeObj *time.Time
+
+	if startTime != "" {
+		layout := "2006-01-02T15:04:05Z"
+
+		parsedTime, err := time.Parse(layout, startTime)
```

Comment:
Should we be using `time.RFC3339` here as that's what we use in the go-sdk, or is the `2006-01-02T15:04:05Z07:00` format a little bit too lengthy to write out on the CLI?

Reply:
The API docs call for ISO8601 which in go requires a layout variable as it differs slightly from RFC3339.  The go-sdk expects input to be a time.Time object but since we're taking text input in the CLI this ensures that ISO8601 is properly supported as detailed in the API documentation.

Reply:
Ahh my bad, I overlooked we were just using that in the tests